packages/components/components/elvis-slider/src/react/elvia-slider.tsx

```diff
@@ -4,6 +4,7 @@ import {
   FormFieldInputSuffixText,
   useInputModeDetection,
 } from '@elvia/elvis-toolbox';
+import { useLanguage } from '@elvia/elvis-toolbox';
```

Comment:
Imports from the same path can be combined. So, `useLanguage` can be included in the import being made on line 6